hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsInputStream.java

```diff
@@ -38,6 +38,10 @@
 import org.apache.hadoop.fs.azurebfs.contracts.exceptions.AzureBlobFileSystemException;
 import org.apache.hadoop.fs.azurebfs.utils.CachedSASToken;
 
+import static java.lang.Math.max;
```

Comment:
any reason for not impoting java.lang.math?

Reply:
When static import is used, I can simply write min(a,b) instead of Math.min(a, b)
Also I don't want to import the whole class statically for just these 2 methods.